src/main/java/org/veupathdb/service/eda/compute/plugins/alphadiv/AlphaDivPlugin.java

```diff
@@ -51,13 +51,13 @@ protected void execute() {
       connection.voidEval(util.getVoidEvalFreadCommand(INPUT_DATA, computeInputVars));
 
       connection.voidEval("alphaDivDT <- alphaDiv(" + INPUT_DATA + ", " + 
-                                                    util.singleQuote(computeEntityIdColName) + ", " + 
-                                                    util.singleQuote(method) + ")");
+                                                    PluginUtil.singleQuote(computeEntityIdColName) + ", " +
+                                                    PluginUtil.singleQuote(method) + ")");
       String dataCmd = "readr::format_tsv(alphaDivDT)";
       String metaCmd = "getMetadata(alphaDivDT)";
 
       getWorkspace().writeDataResult(connection.eval(dataCmd).asString());
-      getWorkspace().writeMetaResult(connection.eval(metaCmd).asString());
+      getWorkspace().writeStatisticsResult(connection.eval(metaCmd).asString());
```

Comment:
Why was this changed from meta to stats?

Reply:
Yeah, this looks like a bug.  Does this work?